src/devtran.cpp

```diff
@@ -616,9 +617,12 @@ Rcpp::List DEVTRAN(const Rcpp::List parin,
       }
       
       if(!this_rec->is_lagged()) {
-        prob.table_call();
+        if(call_event) {
+          prob.event_call();
+        } else {
+          prob.table_call();
+        }
       }
```

Comment:
This is the first work around to maintain event code in `$EVENT` and `$TABLE`.  Basically, if the user has `$EVENT` block, then call that one here; otherwise call `$TABLE`.  Down below, we'll call `$TABLE` if `$EVENT` was called and do nothing if it wasn't.